tools/eos-content-merge

```diff
@@ -4,12 +4,14 @@
 from collections import OrderedDict
 from configparser import ConfigParser
 import json
+from libeosshellcontent import ShellContent
 import os
 import re
 import shutil
 import sys
 
-CONTENT_FILE = '/usr/share/eos-shell-content/content.json'
+CONTENT_DIR = os.getcwd()
+CONTENT_FILE = CONTENT_DIR + '/content/Default/apps/content.json'
```

Comment:
How does this work? It looks like you've changed this tool so that it only runs from an eos-shell-content checkout, but this is intended to run from the installed eos-shell-content-dev. Unless I'm missing something, this looks broken.